tests/__snapshots__/test_video.ambr

```diff
@@ -1,4 +1,65 @@
+  dict({
+    'nullable_content_details': dict({
+      'definition': <VideoDefinition.HD: 'hd'>,
+      'dimension': <VideoDimension.D2: '2d'>,
+      'licensed_content': True,
+      'projection': <VideoProjection.RECTANGULAR: 'rectangular'>,
+      'raw_caption': 'true',
+      'raw_duration': 'PT21M3S',
+    }),
+    'nullable_snippet': dict({
+      'channel_id': 'UCAuUUnT6oDeKwE6v1NGQxug',
+      'channel_title': 'TED',
+      'default_audio_language': 'en',
+      'default_language': 'en',
+      'description': 'Body language affects how others see us, but it may also change how we see ourselves. Social psychologist Amy Cuddy argues that "power posing" -- standing in a posture of confidence, even when we don\'t feel confident -- can boost feelings of confidence, and might have an impact on our chances for success. (Note: Some of the findings presented in this talk have been referenced in an ongoing debate among social scientists about robustness and reproducibility. Read Amy Cuddy\'s response here: http://ideas.ted.com/inside-the-debate-about-power-posing-a-q-a-with-amy-cuddy/)Get TED Talks recommended just for you! Learn more at https://www.ted.com/signup.The TED Talks channel features the best talks and performances from the TED Conference, where the world\'s leading thinkers and doers give the talk of their lives in 18 minutes (or less). Look for talks on Technology, Entertainment and Design -- plus science, business, global issues, the arts and more.Follow TED on Twitter: http://www.twitter.com/TEDTalksLike TED on Facebook: https://www.facebook.com/TEDSubscribe to our channel: https://www.youtube.com/TED',
+      'live_broadcast_content': <LiveBroadcastContent.NONE: 'none'>,
+      'published_at': datetime.datetime(2012, 10, 1, 15, 27, 35, tzinfo=TzInfo(0)),
```

Comment:
There seems to be something off with the CI configuration. This should probably be
```suggestion
      'published_at': datetime.datetime(2012, 10, 1, 15, 27, 35, tzinfo=TzInfo(UTC)),
```